prov/util/src/util_av.c

```diff
@@ -550,10 +550,20 @@ static int util_av_init(struct util_av *av, const struct fi_av_attr *attr,
 			const struct util_av_attr *util_attr)
 {
 	int *entry, i, ret = 0;
+	size_t max_count, param;
+
+	fi_param_define((struct fi_provider *)av->prov, "av_addr_limit",
+			FI_PARAM_SIZE_T, "Defines the maximum number of "
+			"addresses that can be inserted into the AV");
```

Comment:
The parameter should be defined once, not once per av that is created (which is usually only one, but still).

Reply:
Once per provider? How we can check it?

Probably, we can define this as a core run-time parameter like `FI_LOG_LEVEL`, what do you think?

Reply:
A core parameter makes sense to me.  Alternatively, I would move the fi_param_define() into some other init call that isn't associated with an object that can be created multiple times.

Reply:
Moved to core code